src/bootstrap/build/channel.rs

```diff
@@ -32,7 +41,8 @@ pub fn collect(build: &mut Build) {
         }
     }
 
-    // FIXME: this is duplicating makefile logic
```

Comment:
Ah yeah this is just gonna stick around until the makefiles go away, so no need to keep the FIXME